From c8c1d2bf34d140a732ed112b87f8fceaf4d1af55 Mon Sep 17 00:00:00 2001
From: Alan Pevec <alan.pevec@redhat.com>
Date: Tue, 12 Jan 2016 23:18:38 +0100
Subject: [PATCH] Dummy change to trigger OPM build

BTW this docs seems to be out of date, gerrithub is used now
at least for upstream-* branches:
https://review.gerrithub.io/#/q/project:redhat-openstack/openstack-puppet-modules

Also master and stable/* are pushed directly to github so their replicas
on gerrithub are out of sync!

Change-Id: I047001103356796a52d648f3165dd91457c026ef
---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index b080664..f9852e3 100644
--- a/README.md
+++ b/README.md
@@ -33,3 +33,4 @@ to the commit message, push the branch and [initiate a pull request](https://hel
 
     git commit --amend
     git push --set-upstream origin collectd
+
